### PR TITLE
Add upgrade post check for destructive cloud tests

### PR DESCRIPTION
**Pull request by liangxia**
/cc @sunzhaohua2 @jhou1 @JianLi-RH @dis016 @pruan-rht

### CODE REVIEW

File: ci-operator/step-registry/cucushift/upgrade/check/cucushift-upgrade-check-commands.sh [hunk 1]
```diff
@@ -34,10 +34,12 @@ if [ -z "${UPGRADE_SKIP_TAGS}" ] ; then
 else
     export UPGRADE_SKIP_TAGS="${UPGRADE_SKIP_TAGS} and not @customer and not @security"
 fi
+UPGRADE_SKIP_TAGS_CLOUD="${UPGRADE_SKIP_TAGS/and not @destructive/}"
```
Comment (jhou1) [inline ci-operator/step-registry/cucushift/upgrade/check/cucushift-upgrade-check-commands.sh:37]: Thanks, @liangxia 

How does `${UPGRADE_SKIP_TAGS/and not @destructive/}` translate?

Comment (liangxia) [reply ci-operator/step-registry/cucushift/upgrade/check/cucushift-upgrade-check-commands.sh:37]: It removes `and not @destructive` from the default UPGRADE_SKIP_TAGS

File: ci-operator/step-registry/cucushift/upgrade/check/cucushift-upgrade-check-commands.sh [hunk 2]
```diff
@@ -34,10 +34,12 @@ if [ -z "${UPGRADE_SKIP_TAGS}" ] ; then
...
 
 cd verification-tests
 set -x
 cucumber --tags "${UPGRADE_CHECK_RUN_TAGS} and ${UPGRADE_SKIP_TAGS}" -p junit || true
+cucumber --tags "${UPGRADE_CHECK_RUN_TAGS} and ${UPGRADE_SKIP_TAGS_CLOUD} and @cloud and @destructive" -p junit || true
```
Comment (jhou1) [inline ci-operator/step-registry/cucushift/upgrade/check/cucushift-upgrade-check-commands.sh:42]: @sunzhaohua2 Can we restrict it to only the cloud scenario by `-n SCENARIO_NAME`? `@cloud and @destructive` gives us test to autoscaler and health check.

Comment (liangxia) [reply ci-operator/step-registry/cucushift/upgrade/check/cucushift-upgrade-check-commands.sh:42]:
```
UPGRADE_CHECK_RUN_TAGS='@upgrade-check and @4.12 and @aws-ipi and not @fips'
UPGRADE_SKIP_TAGS_CLOUD='not @customer and not @flaky and not @inactive and not @prod-only and not @qeci and not @security and not @stage-only'
```

`cucumber --tags "${UPGRADE_CHECK_RUN_TAGS} and ${UPGRADE_SKIP_TAGS_CLOUD} and @destructive"` return 7 scenarios.
If we add `and @cloud`, the scenario number is controllable.

Comment (liangxia) [reply ci-operator/step-registry/cucushift/upgrade/check/cucushift-upgrade-check-commands.sh:42]: @sunzhaohua2   Could you help add tag `@cloud` to the test cases you want to execute for the post upgrade in verification-tests and cucushift repository ?  Thanks.

Comment (sunzhaohua2) [reply ci-operator/step-registry/cucushift/upgrade/check/cucushift-upgrade-check-commands.sh:42]: https://github.com/openshift/verification-tests/pull/3259
```
$ bundle exec cucumber --tags '@upgrade-check and @4.12 and @aws-ipi and not @fips and not @customer and not @flaky and not @inactive and not @prod-only and not @qeci and not @security and not @stage-only and @cloud' --dry-run
1 scenario (1 skipped)
```

**Pull request merged**